drupal/sites/all/modules/antistatique_article/antistatique_article.info

```diff
@@ -6,7 +6,7 @@ project = antistatique_article
 dependencies[] = antistatique_basic_page
 dependencies[] = disqus
 dependencies[] = field_group
-dependencies[] = file_entity
```

Comment:
It's still used yes. This is the module that is transforming files into "nodes". --> `/admin/content/file`